pkg/server/servicebroker/apiserver_test.go

```diff
@@ -23,8 +23,15 @@ func mockALMBroker(ctrl *gomock.Controller) *ALMBroker {
 func TestValidateBrokerAPIVersion(t *testing.T) {
 	ctrl := gomock.NewController(t)
 	defer ctrl.Finish()
-	err := mockALMBroker(ctrl).ValidateBrokerAPIVersion("n/a")
-	require.EqualError(t, err, "not implemented")
+	// test bad version
+	brokerMock := mockALMBroker(ctrl)
```

Comment:
Can these be done without the operatorclient mocks? It's just more work to remove operator client if we use them in new code

Reply:
These use whatever is in the generated clientsets:
https://github.com/operator-framework/operator-lifecycle-manager/pull/345/files/d9dd9c928c52c1d3c32ff2a9cff4717f63da804e#diff-3a028458a1ff8c8a41758933f6670271L9

Do those need to be updated?

Reply:
nope, that's good - I just saw mock and assumed it was using the MockOperatorClient!